src/main/views/courts/tabs/generalContent.njk

```diff
@@ -129,8 +126,7 @@
   </span>
   <span class="govuk-visually-hidden">{{mceMsg}}</span>
   <textarea class="govuk-textarea urgent-notice-rich-editor" id="urgent-notice" name="alert"
-            rows="2" maxlength="250">{{ generalInfo.alert }}</textarea>
-    {{ tinyMCEAttribution() }}
+            rows="5" maxlength="250">{{ generalInfo.alert }}</textarea>
```

Comment:
Confirm for me please that we want 5 rows instead of 2 now :) Likely the ticket says so, but want to double check.